cli/app.py

```diff
@@ -63,9 +51,12 @@ def create(cls, app_name, config, args=None):
         """
         settings_dir = config.get('defaults', 'settings')
 
-        if app_name in ["provisioner", ]:
+        if app_name not in ["provisioner", "installer"]:
+            raise exceptions.IRUnknownApplicationException(
+                "Application is not supported: '{}'".format(app_name))
```

Comment:
spec is not supported is much more understandable for a user perspective than a new definition "App".